### ISSUE TITLE
Destination BigQuery: map "airbyte_type": "big_number" to BIGNUMERIC

### ISSUE THREAD

**Issue by yurii-bidiuk** [airbytehq/airbyte#12658]
Due to new updates in the building of JSON schema, a new property `"airbyte_type"` was added to the field description. 
One of the possible values for `"airbyte_type"` is `"big_number"`. Now `"airbyte_type"` is ignored and all `"big_number"` fields are mapped as `"STRING"` in destination BigQuery. 
According to #11090 and this [comment](https://github.com/airbytehq/airbyte/issues/11090#issuecomment-1119113122) all fields with `"airbyte_type": "big_number"` should be mapped as `BIGNUMERIC` in destination BigQuery. 

**Input JSON schema example:**
```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "type": "object",
  "properties": {
    "type_big_number": {"type": "string", "airbyte_type": "big_number"},
  }
}
```
**Input data example:**
```
{
  "type_big_number": "1000000000000000000000000000000000.1234"
}
```

**Comment by yurii-bidiuk** [comment]
As datatype mapping for destiantion-bigquery should be done on the normalization side, I'm moving this ticket to the board of Phyton team 
cc: @alexandr-shegeda

**Comment by etsybaev** [comment]
Basically should be closed along the way once this PR is merged https://github.com/airbytehq/airbyte/pull/19721

**Comment by grishick** [comment]
merged into the feature branch
